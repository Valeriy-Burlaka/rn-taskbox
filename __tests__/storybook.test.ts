/**
 * @jest-environment jsdom
 */

import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  framework: 'react-native',
  configPath: './storybook-ondevice/index.js',
});
