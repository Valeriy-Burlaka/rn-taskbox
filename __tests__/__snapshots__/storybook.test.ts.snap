// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button default 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "transparent",
        "flex": 1,
      },
      {
        "backgroundColor": "white",
      },
    ]
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "borderColor": "coral",
        "borderStyle": "solid",
        "borderWidth": 1,
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "violet",
          "borderRadius": 6,
          "flex": 1,
          "justifyContent": "center",
          "maxHeight": 80,
          "maxWidth": 160,
          "opacity": 1,
          "paddingHorizontal": 20,
          "paddingVertical": 20,
        }
      }
    >
      <Text
        style={
          [
            {
              "color": "#000",
            },
            [
              {
                "fontSize": 18,
                "letterSpacing": 1.25,
                "textTransform": "uppercase",
              },
              {
                "fontFamily": "space-mono",
              },
            ],
          ]
        }
      >
        click me
      </Text>
    </View>
  </View>
</View>
`;

exports[`Storyshots Button withPinkBackground 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "transparent",
        "flex": 1,
      },
      {
        "backgroundColor": "white",
      },
    ]
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "borderColor": "coral",
        "borderStyle": "solid",
        "borderWidth": 1,
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "pink",
          "borderRadius": 6,
          "flex": 1,
          "justifyContent": "center",
          "maxHeight": 80,
          "maxWidth": 160,
          "opacity": 1,
          "paddingHorizontal": 20,
          "paddingVertical": 20,
        }
      }
    >
      <Text
        style={
          [
            {
              "color": "#000",
            },
            [
              {
                "fontSize": 18,
                "letterSpacing": 1.25,
                "textTransform": "uppercase",
              },
              {
                "fontFamily": "space-mono",
              },
            ],
          ]
        }
      >
        hey
      </Text>
    </View>
  </View>
</View>
`;

exports[`Storyshots Button withText 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "transparent",
        "flex": 1,
      },
      {
        "backgroundColor": "white",
      },
    ]
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "borderColor": "coral",
        "borderStyle": "solid",
        "borderWidth": 1,
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "violet",
          "borderRadius": 6,
          "flex": 1,
          "justifyContent": "center",
          "maxHeight": 80,
          "maxWidth": 160,
          "opacity": 1,
          "paddingHorizontal": 20,
          "paddingVertical": 20,
        }
      }
    >
      <Text
        style={
          [
            {
              "color": "#000",
            },
            [
              {
                "fontSize": 18,
                "letterSpacing": 1.25,
                "textTransform": "uppercase",
              },
              {
                "fontFamily": "space-mono",
              },
            ],
          ]
        }
      >
        hello
      </Text>
    </View>
  </View>
</View>
`;

exports[`Storyshots Task archived 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "transparent",
        "flex": 1,
      },
      {
        "backgroundColor": "white",
      },
    ]
  }
>
  <View
    style={
      {
        "backgroundColor": "#26c6da",
        "color": "#333",
        "flex": 1,
        "fontSize": 16,
      }
    }
  >
    <RCTSafeAreaView
      emulateUnlessSupported={true}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "white",
          "flexDirection": "row",
          "flexWrap": "nowrap",
          "height": 48,
          "justifyContent": "space-around",
          "paddingHorizontal": 20,
          "width": "100%",
        }
      }
    >
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <Text />
      </View>
      <TextInput
        editable={false}
        placeholder="Input Title"
        style={
          {
            "color": "#aaa",
            "fontFamily": "NunitoSans",
            "fontSize": 14,
            "fontStyle": "normal",
            "lineHeight": 20,
            "padding": 10,
            "width": "95%",
          }
        }
        value="Test Task"
      />
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <Text />
      </View>
    </RCTSafeAreaView>
  </View>
</View>
`;

exports[`Storyshots Task default 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "transparent",
        "flex": 1,
      },
      {
        "backgroundColor": "white",
      },
    ]
  }
>
  <View
    style={
      {
        "backgroundColor": "#26c6da",
        "color": "#333",
        "flex": 1,
        "fontSize": 16,
      }
    }
  >
    <RCTSafeAreaView
      emulateUnlessSupported={true}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "white",
          "flexDirection": "row",
          "flexWrap": "nowrap",
          "height": 48,
          "justifyContent": "space-around",
          "paddingHorizontal": 20,
          "width": "100%",
        }
      }
    >
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            {
              "backgroundColor": "transparent",
              "borderColor": "#26c6da",
              "borderRadius": 1,
              "borderStyle": "solid",
              "borderWidth": 2,
              "height": 18,
              "width": 18,
            }
          }
        />
      </View>
      <TextInput
        editable={false}
        placeholder="Input Title"
        style={
          {
            "backgroundColor": "transparent",
            "fontFamily": "NunitoSans",
            "fontSize": 14,
            "fontStyle": "normal",
            "lineHeight": 20,
            "padding": 10,
            "width": "95%",
          }
        }
        value="Test Task"
      />
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <Text />
      </View>
    </RCTSafeAreaView>
  </View>
</View>
`;

exports[`Storyshots Task pinned 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "transparent",
        "flex": 1,
      },
      {
        "backgroundColor": "white",
      },
    ]
  }
>
  <View
    style={
      {
        "backgroundColor": "#26c6da",
        "color": "#333",
        "flex": 1,
        "fontSize": 16,
      }
    }
  >
    <RCTSafeAreaView
      emulateUnlessSupported={true}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "white",
          "flexDirection": "row",
          "flexWrap": "nowrap",
          "height": 48,
          "justifyContent": "space-around",
          "paddingHorizontal": 20,
          "width": "100%",
        }
      }
    >
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            {
              "backgroundColor": "transparent",
              "borderColor": "#26c6da",
              "borderRadius": 1,
              "borderStyle": "solid",
              "borderWidth": 2,
              "height": 18,
              "width": 18,
            }
          }
        />
      </View>
      <TextInput
        editable={false}
        placeholder="Input Title"
        style={
          {
            "backgroundColor": "transparent",
            "fontFamily": "NunitoSans",
            "fontSize": 14,
            "fontStyle": "normal",
            "lineHeight": 20,
            "padding": 10,
            "width": "95%",
          }
        }
        value="Test Task"
      />
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <Text />
      </View>
    </RCTSafeAreaView>
  </View>
</View>
`;
